<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewUser</class>
 <widget class="QCDialog" name="QNewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>250</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>NewUser</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>22</y>
     <width>44</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>51</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_commnet">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QCPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCPushButton" name="btn_sure">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Sure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>164</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comb_auth">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>117</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>18</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>89</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>55</y>
     <width>69</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>121</y>
     <width>69</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Authority:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_confirm">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>84</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCPushButton</class>
   <extends>QPushButton</extends>
   <header>qcpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCDialog</class>
   <extends>QWidget</extends>
   <header>qcdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
